src/appleseed/renderer/modeling/frame/frame.h

```diff
@@ -81,7 +81,7 @@ class APPLESEED_DLLSYMBOL Frame
     // Print frame settings to the renderer's global logger.
     void print_settings();
 
-    // Return the camera name.
+    // Return the name of the active camera.
     const char* get_camera_name() const;
```

Comment:
Now I realize that we might want to also rename `get_camera_name()` to `get_active_camera_name()`.